bokeh/models/annotations.py

```diff
@@ -1096,6 +1097,42 @@ class Whisker(Annotation):
 
     level = Override(default="underlay")
 
+class InfoPane(Annotation):
+    ''' Renders a info-pane.
+
+    .. note::
+        This model is currently managed by BokehJS and is not useful
+        directly from python.
+
+    '''
+    level = Override(default="overlay")
+
+    anchor = Enum(TooltipAttachment, help="""
+    Whether the pane should be anchored at the left or right of the pane
+    or above or below it, or if it should be automatically placed
+    in the horizontal or vertical dimension.
+    """)
+
+    x_anchor = Float(default=0, help="""
+    Initial anchor value for pane to apply to the x-coordinate.
+    """)
+
+    y_anchor = Float(default=0, help="""
+    Initial anchor value for pane to apply to the y-coordinate.
+    """)
```

Comment:
I also think these should just be `x` and `y` and not `x_anchor` and `y_anchor`. And the reason is that the "anchor" refers to a position relative to the `InfoPane` itself. I.e. when the location is `x=10`, `y=20` the anchor point of the info pane specifies how the pane should be draw relative to that location.

Reply:
We will also need to consider handling categorical coordinates at some point but I am OK skipping that for now and opening an issue for later.

Reply:
Yeah making them as` x_anchor` and `y_anchor` seems a bit misleading. I will change it to just x and y.